pkgs/tools/security/magnifier/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, llvmPackages_14
```

Comment:
Please use `override` or similar in `all-packages.nix` to pin LLVM to a specific version.

Reply:
I think I understand what you mean, but not how to do it. I did try to grep for any other packages using override to pin an llvm version as it must be common. `pkgs/development/compilers/rust/1_63.nix` must be doing something like what you intend but I don't really get what to copy.

Reply:
My bad, I phrased that cryptically. You basically need to do this:
```nix
   magnifier = callPackage ../tools/security/magnifier {
     llvmPackages = llvmPackages_14;
   };
```

Reply:
ah, thanks! fixed now